src/renderers/shaders/ShaderLib.js

```diff
@@ -93,7 +93,9 @@ THREE.ShaderLib = {
 				"emissive" : { type: "c", value: new THREE.Color( 0x000000 ) },
 				"roughness": { type: "1f", value: 0.5 },
 				"metalness": { type: "1f", value: 0 },
-				"envMapIntensity" : { type: "1f", value: 1 } // temporary
+				"envMapIntensity" : { type: "1f", value: 1 }, // temporary
+				"clearCoat": { type: "1f", value: 0 },
+				"clearCoatRoughness": { type: "1f", value: 0 }
```

Comment:
These new properties are not properties of `StandardMaterial`, and belong in `THREE.ShaderLib[ 'physical' ]`. In fact, there is already a "future" placeholder for them there. :)